src/content/docs/plugins/submission.md

```diff
@@ -2,3 +2,96 @@
 title: Plugin Submission
 description: How to submit your plugin to the official Vencord repository
 ---
+
+Submitting a plugin to Vencord allows you to share your work with the community and have it included in the official Vencord releases. Follow this guide to submit your plugin for consideration.
+
+## Prerequisites
+
+Before submitting your plugin, ensure that:
+- Your plugin is fully developed and tested (refer to the [Creating Plugins guide](/plugins/creating) for help).
+- You have a GitHub account and are familiar with basic Git commands.
+- Your plugin adheres to the code quality, structure, and naming conventions used in Vencord (e.g., camelCase for plugin folder names).
+
+## Submission Process
+
+The submission process involves forking the Vencord repository, adding your plugin, and creating a pull request for review by the maintainers.
+
+### Step 1: Fork the Vencord Repository
+
+1. Go to the official [Vencord GitHub repository](https://github.com/Vendicated/Vencord).
+2. Click the **Fork** button in the top-right corner to create a copy of the repository under your GitHub account.
+3. Clone the forked repository to your local machine:
+   ```bash
+   git clone https://github.com/YourUsername/Vencord
+   cd Vencord
+   ```
+
+### Step 2: Add Your Plugin
+
+1. Navigate to the `src/plugins` folder in your cloned repository:
+   ```bash
+   cd src/plugins
+   ```
+2. Create a new folder for your plugin using camelCase (e.g., `myFirstPlugin`):
+   ```bash
+   mkdir myFirstPlugin
+   ```
+3. Add your plugin files (`index.ts`, `README.md`, and optionally `native.ts`):
+    - **`index.ts`**: This is the main entry point for your plugin's code.
+    - **`README.md`**: Document your plugin, explain its functionality, and provide installation instructions and usage examples.
```

Comment:
inbuilt plugins do not need installation instructions

Reply:
Done, resolved everything in last commit!